File: programming problems/HackerRank/30 days of code/day 3, intro to conditional statements/day_3_intro_to_conditional_statements.rb
#!/bin/ruby

require 'json'
require 'stringio'

N = gets.strip.to_i

if N % 2 == 1
    puts "Weird"
elsif N >= 2 && N <= 5
    puts "Not Weird"
elsif N >= 6 && N <= 20
    puts "Weird"
elsif N > 20
    puts "Not Weird"
end

File: programming problems/HackerRank/30 days of code/day 3, intro to conditional statements/day_3_intro_to_conditional_statements.py
#!/bin/python3

import math
import os
import random
import re
import sys

if __name__ == '__main__':
    N = int(input().strip())

    if N % 2:
        print("Weird")
    elif N >= 2 and N <= 5:
        print("Not Weird")
    elif N >= 6 and N <= 20:
        print("Weird")
    elif N > 20:
        print("Not Weird")